packages/preview/plotst-0.1.0/README.md

```diff
@@ -0,0 +1,225 @@
+# A plotting library for Typst
+
+A Typst library for drawing graphs and plots.
+Made by Gewi413 and Pegacraffft
+
+## Currently supported graphs
+
+- Scatter plots
+
+- Graph charts
+
+- Histograms
+
+- Bar charts
+
+- Pie charts
+
+- Overlaying plots/charts
+
+  (more to come)
+
+## How to use
+
+To use the package you can import it through this command `import "@preview/plotst:0.1.0": *`.
+The documentation is found in the `Docs.pdf` in the `docs` folder. It contains all functions necessary to use this library. It also includes a tutorial to create every available plot under their respective render methods.
+
+If you need some example code, check out `main.typ` in the `examples` folder. It also includes a compiled version of the current `main.typ`
+
+## Examples:
+
+All these images were created using the `main.typ`
+
+### Scatter plots
+
+```js
+    // Plot 1:
```

Comment:
There seems to be some superfluous indent here. Would be great if the examples were indented consistently with 2 or 4 spaces. Ideally 2 as that is the "official" style.

Reply:
Changed the readme to have consistent indentation. Now it should be fine